### DISCUSSION TITLE
Modernization workflow scan results

### DISCUSSION THREAD

**Discussion by mgmjtech** [pnp/modernization#544] [Q&A] [answered]
Does anyone know why the "workflow scan" results come back with hundreds of workflows with definition ID 0000000-000-0000-0000-0000? What does this mean?

**Comment by mohammadamer** [comment]
Hi!
I have noticed that I got on my scan results only two workflows that have definition ID 00000000-0000-0000-0000-000000000000 and the Version is 2010 but the other workflows that have proper GUID, have version 2013.
I am curious about the reason why those two workflows have this zeros GUID.
I would appreciate help on that

**Comment by jansenbe** [comment, marked as answer]
If a workflow is created on the list as unique workflow for that list it does not always have "definition" in the API, hence the empty value.
